src/index.js

```diff
@@ -11,6 +11,10 @@ const http2 					= require("http2");
 const https 					= require("https");
 const queryString 				= require("qs");
 const os 						= require("os");
+
+// Disable global colors logs
+if(String(process.env.COLORS).toLowerCase() === "false") process.env.FORCE_COLOR = 0;
```

Comment:
I don't understand the concept, could you explain why need a `COLORS` env var to set another `FORCE_COLOR` env var?

Reply:
If it's only control the kleur maybe you should send this PR to the [kleur project](https://github.com/lukeed/kleur/blob/master/index.mjs#L10).

Reply:
I was wrong. `COLORS` is `MOL_LOGGER__OPTIONS__COLORS` should be for compatibility with env of moleculer-runner.

Problem:
![photo_2020-10-27_21-20-40](https://user-images.githubusercontent.com/451306/97345042-a4921780-189a-11eb-811c-895f5b79601d.jpg)

What is expected:
![photo_2020-10-27_21-20-33](https://user-images.githubusercontent.com/451306/97345088-b4116080-189a-11eb-8d0a-cb6dd9e1de1f.jpg)